packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -419,6 +420,27 @@ export class ApolloServerBase {
         } => typeof maybeTaggedServerListener.serverListener === 'object',
       );
 
+      taggedServerListeners.forEach(
+        ({ serverListener: { schemaDidLoadOrUpdate } }) => {
+          if (schemaDidLoadOrUpdate) {
+            try {
+              schemaManager.onSchemaLoadOrUpdate(schemaDidLoadOrUpdate);
+            } catch (_) {
+              // TODO: Once a gateway version providing the core schema to
```

Comment:
If this is something that needs to be fixed before the PR can be merged, include "FIXME" so we don't forget.

Reply:
Yep, this is something we should fix prior to merge (once we release gateway that has `onSchemaLoadOrUpdate()`).

I've also fixed a bug in the error message here (the name of the hook was wrong).